frontend/src/credit_transfers/components/CreditTransactionsPage.js

```diff
@@ -8,20 +8,42 @@ import { DEFAULT_ORGANIZATION } from '../../constants/values';
 import history from '../../app/History';
 import Loading from '../../app/components/Loading';
 import CreditTransferTable from './CreditTransferTable';
+import {organization} from "../../sampleData";
```

Comment:
Is this a sample test code?

Reply:
It's an unused import. I'll clean it up.